.pre-commit-config.yaml

```
@@ -0,0 +1,77 @@
+  - repo: https://github.com/igorshubovych/markdownlint-cli
+    rev: v0.39.0
+    hooks:
+      - id: markdownlint
+        args: ['--fix']
+
+# Configuration for specific hooks
+ci:
+  autofix_commit_msg: 'style: auto fixes from pre-commit hooks'
+  autofix_prs: true
+  autoupdate_commit_msg: 'chore: pre-commit autoupdate'
+  autoupdate_schedule: weekly
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)